Makefile

```diff
@@ -746,8 +740,15 @@ bootstrap-device: bootstrap
 	rm -f $(BUILD_STAGE)/.fakeroot_$$(echo $@ | rev | cut -f2- -d"-" | rev)
 	touch $(BUILD_STAGE)/.fakeroot_$$(echo $@ | rev | cut -f2- -d"-" | rev)
 	mkdir -p $(BUILD_DIST)
-%-install:
-	+TO_INSTALL=1 $(MAKE) $(shell echo $@ | $(SED) 's/-install//')-package
+%-install:: export BUILD_DIST=$(BUILD_ROOT)/build_dist/$(MEMO_TARGET)/$(MEMO_CFVER)/stageinstall
+%-install: %
+	+$(MAKE) $$(echo $@ | $(SED) 's/-install//')-package
+	temp=$$(ssh -p $(MEMO_DEVICE_PORT) root@$(MEMO_DEVICE_IP) 'mktemp -d'); \
+	rsync -z -e "ssh -p $(MEMO_DEVICE_PORT)" $(BUILD_DIST)/*.deb root@$(MEMO_DEVICE_IP):$$temp; \
+	ssh -p $(MEMO_DEVICE_PORT) root@$(MEMO_DEVICE_IP) 'apt-get update'; \
+	ssh -p $(MEMO_DEVICE_PORT) root@$(MEMO_DEVICE_IP) "apt-get reinstall $$temp/*.deb"; \
```

Comment:
Pretty sure `apt-get reinstall` isn't a valid operation.

Reply:
It is, just double-checked.